src/App.jsx

```diff
@@ -3,14 +3,17 @@ import Courses from './components/Courses/Courses';
 import Login from './components/Login/Login';
 import Registration from './components/Registration/Registration';
 import CourseInfo from './components/CourseInfo/CourseInfo';
-import CreateCourse from './components/CreateCourse/CreateCourse';
+import CourseForm from './components/CourseForm/CourseForm';
 import './App.css';
 import { Switch, Route } from 'react-router-dom';
 import { BrowserRouter } from 'react-router-dom';
 import React, { useState } from 'react';
+import store from './store';
+import { PrivateRoute } from './components/PrivateRouter/PrivateRouter';
 
 const check = () => {
```

Comment:
I'd recommend to refactor it a bit: use `useSelector` to retrieve token, as this would be more reliable way to check if user is logged in. By dispatching `clearAuth` action you can clear stored user data